NGitLab.Mock/NGitLab.Mock.csproj

```diff
@@ -1,14 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;net6.0</TargetFrameworks>
```

Comment:
Why? Is this needed to consume the latest LibGit2Sharp lib?

As a reminder, to accommodate the most targets possible we chose `net461` and `netsatdandard2.0`, in accordance with what Microsoft suggests [here](https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting):

> ✔️ DO start with including a netstandard2.0 target.
>
> Most general-purpose libraries should not need APIs outside of .NET Standard 2.0. .NET Standard 2.0
> is supported by all modern platforms and is the recommended way to support multiple
> platforms with one target.

> ✔️ CONSIDER adding a target for net461 when you're offering a netstandard2.0 target.
> 
> Using .NET Standard 2.0 from .NET Framework has some issues that were addressed in .NET
> Framework 4.7.2. You can improve the experience for developers that are still on .NET Framework
> 4.6.1 - 4.7.1 by offering them a binary that is built for .NET Framework 4.6.1.

Reply:
> Is this needed to consume the latest LibGit2Sharp lib?

I don't using all outdated dependencies is really an option.

`net461` is not supported anymore, I think it's fine and it's only the mock project.